### PR TITLE
Describe REST API 

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  
+## Stop
+
+* **URL**
+
+  /v1/_stop
+  
+* **Method:**
+
+  `GET` | `PUT`
```
Comment (afiskon) [inline API.md]: Actually GET /v1/_stop does nothing.

File: API.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  
+* **Success Response:**
+  
+  * **Code:** `200` <br />
+  
+* **Error Response:**
+
+  * **Code:** `404` <br />
+
+* **Sample Call:**
```
Comment (afiskon) [inline API.md]: Better call it "Examples"

File: API.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  
+  some_value
+  
+* **Success Response:**
+  
+  * **Code:** `200` <br />
+  
+* **Error Response:**
+
+  * **Code:** `404` <br />
```
Comment (afiskon) [inline API.md]: Actually 404 is not an error. It's a regular response that means that requested key doesn't exist.

File: API.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  
+* **Data Params**
+
+  Stored value.
+  
+  **Optional:**
+  
+  some_value
+  
+* **Success Response:**
```
Comment (afiskon) [inline API.md]: s/Success/Successful

File: API.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  
+## KV
+
+* **URL**
+
+  /v1/kv/key
+  
+* **Method:**
+
+  `GET` | `DELETE` | `PUT`
```
Comment (afiskon) [inline API.md]: Well, in fact it's 3 separated requests. GET /v1/kv/some_key reads a value, DELETE deletes it and PUT creates or updates a value.

File: API.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  
+## Stop
+
+* **URL**
+
+  /v1/_stop
+  
+* **Method:**
+
+  `PUT`
```
Comment (Alexey-N-Chernyshov) [inline API.md]: fixed

File: API.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
+## Index
+
+
+* **URL**
+
+  /
+
```
Comment (Alexey-N-Chernyshov) [inline API.md]: вавава

**Review by afiskon — CHANGES_REQUESTED** [review]

File: API.md (hunk 5 above)
Comment (Alexey-N-Chernyshov) [reply API.md]: done

File: API.md (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply API.md]: done

**Comment by afiskon** [comment]
LGTM. Merged. Thank you a lot!

**Pull request merged**